embc-app/ClientApp/src/app/core/services/registration.service.ts

```diff
@@ -61,4 +61,31 @@ export class RegistrationService extends RestService {
         catchError(this.handleError),
       );
   }
+
+  async printReferrals(registrationId: string, referralIds: string[], addSummary: boolean): Promise<void> {
+    const blob = await this.getReferralPdfs(registrationId, referralIds, addSummary);
+
+    // check if IE, Edge, etc
```

Comment:
Comment about comments. It isn't immediately obvious which is the IE and which is the good browser logic without knowing what msSaveBlob is. Comments won't hurt.

Reply:
Please see updated code and comments.

Reply:
Great. This is much more explanatory of the reason for the logic.